.readme-partials.yml

```diff
@@ -132,3 +132,24 @@ body: |-
     [LOGGING_TRACE_KEY]: 'custom-trace-value'
   }, 'Bunyan log entry with custom trace field');
   ```
+
+  ### Error handling with a default callback
+  The `LoggingBunyan` class creates an instance of `Logging` which creates the `Log` class from `@google-cloud/logging` package to write log entries. 
+  The `Log` class writes logs asynchronously and there are cases when log entries cannot be written and an error is 
+  thrown.  If the error is not handled properly, it could crash the application. One possible way to handle the error is to provide a default callback
```

Comment:
nit: I suggest to provide all the ways to handle the error not only the new one that you introduced.

Reply:
I dont think there are any ways to handle such errors, except of using global handlers like:
'''js
process.setUncaughtExceptionCaptureCallback(console.error);
'''
There is a work item on bunyan library for this issue: [log functions should offer a slot for callback function](https://github.com/trentm/node-bunyan/issues/95)